jmx-metrics/src/main/groovy/io/opentelemetry/contrib/jmxmetrics/GroovyMetricEnvironment.java

```diff
@@ -70,16 +69,24 @@ public GroovyMetricEnvironment(
                 System.setProperty(key, value.toString());
               }
             });
-        // this call will dynamically load the autoconfigure extension
-        // and take care of provider and exporter creation for us based on system properties.
-        GlobalOpenTelemetry.get();
-        meterProvider = (SdkMeterProvider) GlobalMeterProvider.get();
+        // call the autoconfigure extension and take care of provider and exporter creation for us
```

Comment:
Do you feel like migrating the above lines to use `addPropertiesSupplier`?

Reply:
Good idea 👍